components/command/CommandPanel.vue

```diff
@@ -64,6 +64,9 @@ const result = $computed<QueryResult>(() => commandMode
   : searchResult,
 )
 
+const isMac = useIsMac()
+const modifierKeyName = $computed(() => isMac.value ? 'Command' : 'Ctrl')
```

Comment:
```suggestion
const modifierKeyName = $computed(() => isMac.value ? '⌘' : 'Ctrl')
```

Reply:
That's a good idea.🙂  Here's the new screenshot:

<img width="665" alt="Screenshot 2023-01-29 at 15 50 01" src="https://user-images.githubusercontent.com/1425259/215310106-3bc065b2-cb33-4a9c-b56c-1b80a13a2d57.png">